graphql/resolvers.go

```diff
@@ -1806,7 +1806,7 @@ func (r *mutationResolver) EnqueuePatch(ctx context.Context, patchID string) (*r
 		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error creating new patch: %s", err.Error()))
 	}
 
-	_, err = r.sc.EnqueueItem(restModel.FromStringPtr(newPatch.Project), restModel.APICommitQueueItem{Issue: newPatch.Id}, false)
+	_, err = r.sc.EnqueueItem(restModel.FromStringPtr(newPatch.Project), restModel.APICommitQueueItem{Issue: newPatch.Id, Source: restModel.ToStringPtr(commitqueue.SourceCommandLine)}, false)
```

Comment:
Currently, each method to add to the commit queue only handles a specific type of patch source. In general if there is no pull request in sight, it'll be a CLI patch

Reply:
Technically if we're enqueuing from the UI, it's really it's own "source", right? I wonder if now that we're re-factoring, we can introduce a new source, or use a better source name that "cli" (although that might require a separate ticket).

Reply:
Really what's needed for the logic is differentiating whether the changes come from a pr or diff. I renamed CLI to diff, but it'll be in 2 parts. This part just saves commit queue items with the "diff" source rather than CLI now